app/views/course/achievement/course_users/index.html.slim

```diff
@@ -3,4 +3,16 @@
 
 = render partial: 'achievement', object: @achievement
 
-= render partial: 'course_users_form', locals: { course: @course, achievement: @achievement }
+div.alert.alert-info
+  = simple_format(t('.notice'))
+
+- collection = @course.course_users.with_approved_state.without_phantom_users.students.order_alphabetically
+= render partial: 'course_users_form', locals: { course: @course, achievement: @achievement, collection: collection,
+                                                 css: 'normal', button: 'button' }
+
+br
+h4 = t('.phantom_user_header')
+
+- collection = @course.course_users.with_approved_state.phantom.students.order_alphabetically
+= render partial: 'course_users_form', locals: { course: @course, achievement: @achievement, collection: collection,
+                                                 css: 'phantom', button: 'phantom_button' }
```

Comment:
LGTM, but I lean towards less locals into partials in general (to make it is easier for us to understand one) - don't want to have to manage too many variables.